### ISSUE TITLE
Cannot determine the sync status for Git remote

### ISSUE THREAD

**Issue by lud-wj** [git-town/git-town#2574]
I just upgraded to the latest version on ubuntu, previously I was on 0.7.something.

git version 2.37.2

I have this message when trying to sync on a branch.

```
[feat/tpl-fun] git fetch --prune --tags
panic: cannot determine the sync status for Git remote "[origin/feat/tpl-fun: en avance de 2] fix docs" and branch name "feat/tpl-fun"

goroutine 1 [running]:
github.com/git-town/git-town/v9/src/git.determineSyncStatus({0xc00028004f, 0xc}, {0xc000280073, 0x2e})
        /home/kevlar/git-town/src/git/backend_commands.go:133 +0x209
github.com/git-town/git-town/v9/src/git.ParseVerboseBranchesOutput({0xc000280000, 0x1d0})
        /home/kevlar/git-town/src/git/backend_commands.go:104 +0x1da
github.com/git-town/git-town/v9/src/git.(*BackendCommands).BranchesSyncStatus(0xc000174490)
        /home/kevlar/git-town/src/git/backend_commands.go:81 +0x85
github.com/git-town/git-town/v9/src/execute.LoadBranches(0xc000174480, {0x0?})
        /home/kevlar/git-town/src/execute/load_branches.go:10 +0x6a
github.com/git-town/git-town/v9/src/cmd.determineSyncConfig(0x0, 0xc000174480, 0x0)
        /home/kevlar/git-town/src/cmd/sync.go:107 +0x6b
github.com/git-town/git-town/v9/src/cmd.sync(0x0?, 0x0, 0x0)
        /home/kevlar/git-town/src/cmd/sync.go:69 +0x1c5
github.com/git-town/git-town/v9/src/cmd.syncCmd.func1(0xc0001bc300?, {0xf7f160?, 0x0?, 0x0?})
        /home/kevlar/git-town/src/cmd/sync.go:46 +0x6b
github.com/spf13/cobra.(*Command).execute(0xc0001bc300, {0xf7f160, 0x0, 0x0})
        /home/kevlar/git-town/vendor/github.com/spf13/cobra/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0xc00019c000)
        /home/kevlar/git-town/vendor/github.com/spf13/cobra/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        /home/kevlar/git-town/vendor/github.com/spf13/cobra/command.go:968
github.com/git-town/git-town/v9/src/cmd.Execute()
        /home/kevlar/git-town/src/cmd/core.go:48 +0x6e8
main.main()
        /home/kevlar/git-town/main.go:20 +0x25
```

If I delete everything related to the branch in .git/config it works once.

**Comment by lud-wj** [comment]
Well actually I had deleted my access token yesterday. It did not seem to block on 0.7 but anyway, you can close this as far as I am concerned.

**Comment by kevgo** [comment]
Hi @lud-wj thanks for reporting this. This seems a duplicate of https://github.com/git-town/git-town/issues/2469. This has been fixed recently in https://github.com/git-town/git-town/pull/2478 but the fix hasn't been released yet. This should happen in the next 2 weeks. In the meantime, feel free to use an older release like https://github.com/git-town/git-town/releases/tag/v9.0.0 or https://github.com/git-town/git-town/releases/tag/v8.0.0, which shouldn't have this issue.

I'll close this issue for now. Please comment if you have further problems around this issue. Thanks!

Btw very exciting to see french-speaking users of Git Town 🎉

**Comment by lud-wj** [comment]
Haha that git output log betrayed me :D

**Comment by kevgo** [comment]
FYI the fix for this has been released as [v10.0.0](https://github.com/git-town/git-town/releases/tag/v10.0.0). Please give it a try and let me know if the issue still exists.

**Comment by lud-wj** [comment]
Well I changed my token in `~/.gitconfig` in `[git-town] github-token` but git sync still works now. Which seems logical because only SSH should be needed for those commands.

So it seems to work but I don't know how to really test the error detection when using an invalid token.

Thanks for the update!
